cli/cmd/install.go

```diff
@@ -126,13 +122,30 @@ func newInstallOptions() *installOptions {
 
 func newCmdInstall() *cobra.Command {
 	options := newInstallOptions()
-
 	cmd := &cobra.Command{
-		Use:   "install [flags]",
-		Short: "Output Kubernetes configs to install Linkerd",
-		Long:  "Output Kubernetes configs to install Linkerd.",
+		Use:       "install [admin|user] [flags]",
```

Comment:
I find these names a bit misleading, we're not installing an admin or a user.  I'd expect this to be something more explicit like `linkerd install prepare` and `linkerd install control` or `linkerd install control-plane`.  

But if the shed has sailed on these names then :+1:

Reply:
i'm not strictly tied to admin/user, but we can decide after merge.